Assignment6.Rmd

```diff
@@ -21,49 +13,99 @@ library(rpart)
 #Data
 ```{r}
 #Upload the data sets MOOC1.csv and MOOC2.csv
-M1 <- read.csv("MOOC1.csv", header = TRUE)
-
-M2 <- 
-
+M1 <- 
+M1 <- read.table("MOOC1.csv", sep = ",", header = TRUE)
+ M2 <- 
+M2 <- read.table("MOOC2.csv", sep = ",", header = TRUE)
 ```
 
-#Decision tree
+ #Decision tree
 ```{r}
-#Using the rpart package generate a classification tree predicting certified from the other variables in the M1 data frame. Which variables should you use?
-
-c.tree1 <- 
-
+#Using the rpart package generate a classification tree predicting certified from the other variables in the M1 data frame.
+ c.tree1 <- 
+#we build this model with the factors: forum.posts, grade and assignment
+ c.tree1 <- rpart(certified ~ forum.posts + grade + assignment, method="class", data=M1, control=rpart.control(minsplit = 1, minbucket = 1, cp = 0.0001))
```

Comment:
Good work 👍 It may be necessary to `scale()` your variables so that `forum.posts` does not swamp the other two. `forum.posts` has many more increments than the others and so the algorithm will likely preference it when building the tree